### PR TITLE
Add fitted template to commented out templates in the new card template

### PR DESCRIPTION
**Pull request by FadhlanR**

### CODE REVIEW

File: packages/host/tests/integration/components/operator-mode-test.gts [hunk 1]
```diff
@@ -2575,14 +2576,25 @@ module('Integration | operator-mode', function (hooks) {
...
       await triggerEvent(document, 'mousemove', {
         clientX: targetRect.left + targetRect.width / 2,
-        clientY: targetRect.top - 50,
+        clientY: targetRect.top - marginTop,
       });
 
       await triggerEvent(itemElement, 'mouseup', {
         clientX: targetRect.left + targetRect.width / 2,
-        clientY: targetRect.top - 50,
+        clientY: targetRect.top - marginTop,
```
Comment (FadhlanR) [inline packages/host/tests/integration/components/operator-mode-test.gts:2597]: Changes to this test are unrelated to this PR. However, they fix the failing test on main.

**Review by habdelra — COMMENTED** [review]
This looks good, but I'm not sure about the CI test failure that you are seeing, is it related to your changes?

**Comment by FadhlanR** [comment]
> This looks good, but I'm not sure about the CI test failure that you are seeing, is it related to your changes?

Nope, I've checked it, it comes from main. However, I'm fixing it.

**Comment by habdelra** [comment]
@FadhlanR actually can you back out your fix for that issue (https://github.com/cardstack/boxel/pull/1684/commits/5a9dc0910739735c414828a2b315d23d545d04c4). it's actually coming from the recently merged animations PR (theres some other problems in this branch too). the fix should just not be to run animations in our tests. we have reverted that PR in main, so just take the upstream changes and your branch should be ok.

**Pull request merged**